### ISSUE TITLE
[EssentialsDiscord]  Error occurred while enabling EssentialsDiscord v2.21.0-dev+120-cd06e73

### ISSUE THREAD

**Issue by woodfairy** [EssentialsX/Essentials#5959]
### Type of bug

Compatibility issue, Error in console

### `/ess dump all` output

https://essentialsx.net/dump.html?id=6a474690ab0644f7908706d95e363cb2

### Error log (if applicable)

https://paste.gg/p/anonymous/2f7ed6db04aa4800adcb5466490351ee

### Bug description

It suddenly stopped working. Used to work fine before (broke after updating to latest version, from 1 or 2 builds ago)

### Steps to reproduce

Update from  EssentialsXDiscord-2.21.0-dev+115-6157668.jar

### Expected behaviour

Everything should work as before....

### Actual behaviour

It does not load the plugin because of an exception

**Comment by woodfairy** [comment]
After downgrading to https://ci.ender.zone/job/EssentialsX/1591/artifact/jars/ everything works fine again. 
I also tried this build, which failed as well: https://ci.ender.zone/job/EssentialsX/1595/artifact/jars/

**Comment by GGLinnk** [comment]
Have same bug here : https://essentialsx.net/dump.html?id=d24de2d9867e4b8884c0d2d4710987d4
Updated from : EssentialsXDiscord-2.21.0-dev+73-2cb0af0.jar

**Comment by Giantvince1** [comment]
This issue likely happened as of dev-119 because that's where dependencies that EssDiscord uses were updated, based on changelogs. I also noticed it was the only component touched on that update, and for me, dev-119 is the one that broke. I had been on dev-117 without issue. But the moment I installed dev-119, EssDiscord broke completely, causing EssDiscordLink to also break. As an aside, dev-118 does indeed work; I just grabbed it and started the server. My `/ess dump all` using dev-120: https://essentialsx.net/dump.html?id=6ff65771917b416d8ebd706c727f1ab2

I plan to fork EssX to revert the JDA dependency update to see if that fixes the issue, and if it does, I'll link to my fork here.
